src/central-server-audits.rst

```diff
@@ -25,30 +38,26 @@ As of `version 0.6 <https://github.com/getodk/central/releases/tag/v0.6.0-beta.0
    - Update Submission Data
    - Update Attachments
 
- - **Web User Actions**
-
-   - Create
-   - Update Details (display name, email, password)
-   - Assign Role
-   - Revoke Role
-   - Retire
-
  - **App User Actions**
 
    - Create
-   - Assign Role
-   - Revoke Role
-   - Revoke Access
+   - Give Access
+   - Remove Access
+   - Revoke
    - Delete
 
  - **Public Link Actions**
 
    - Create
-   - Assign Role
-   - Revoke Role
-   - Revoke Access
+   - Give Access
+   - Remove Access
+   - Revoke
    - Delete
 
+ - **System Actions**
```

Comment:
I'd make this match the drop down so people (including future reviewers or docs editors) don't have to think.

```
 - **Server Configuration Actions**
   - Set
```